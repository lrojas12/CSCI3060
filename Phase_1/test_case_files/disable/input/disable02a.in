login
standard
Tarzan
disable
logout
quit