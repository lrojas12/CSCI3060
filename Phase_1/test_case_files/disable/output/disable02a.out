Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Enter Account holder’s name:
You are currently logged in as Tarzan
Bank Account Number: 00001
Balance: $1000.00
Transaction Payment Plan: non-student
Status: active
Mode in which you wish to login as: 
You are currently logged in as an administrator.
ERROR: You need to be an administrator to disable an account.
You have successfully logged out of your account.