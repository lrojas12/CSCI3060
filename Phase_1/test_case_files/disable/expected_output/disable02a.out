Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Enter Account holder’s name:
You are currently logged in as Tarzan
Bank Account Number: 00001
Balance: $01000.00
Transaction Payment Plan: Non-student
Status: Active
Mode in which you wish to login as: 
You are currently logged in as an administrator.
ERROR: You need to be an administrator to disable an account.
You have successfully logged out of your account.